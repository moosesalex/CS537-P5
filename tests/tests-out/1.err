./run-tests.sh: line 27: 2651077 Segmentation fault      (core dumped) ./tests/1.tmp ./test_files/one.txt ./test_files/two.txt ./test_files/three.txt
