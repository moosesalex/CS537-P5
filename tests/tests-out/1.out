RDD has dependencies, executing them first.
RDD has no dependencies, ready to execute.
Done materializing rdd 0x55dae2adf9b0
RDD has no dependencies, ready to execute.
Done materializing rdd 0x55dae2ae0080
one
